src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -240,7 +238,6 @@ def __call__(self, input_ids, token_type_ids, position_ids, attention_mask, dete
         return hidden_states
 
 
-# Copied from transformers.models.bert.modeling_flax_bert.FlaxBertSelfAttention with Bert->BigBird
```

Comment:
This class is the same as FlaxBertSelfAttention with Bert->BigBird: don't think we should remove the `# Copied from...` statement